src/main/kotlin/no/nav/familie/ks/sak/integrasjon/oppgave/OppgaveService.kt

```diff
@@ -220,6 +220,14 @@ class OppgaveService(
         }
     }
 
+    fun oppdaterBehandlingstemaPåOppgaverFraBehandling(
```

Comment:
Man burde nok også sende melding til saksstatistikk ved endring av behandlingstema. Ser ikke ut som vi gjør det.

Reply:
Opprette et eget favro-kort for det her https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=NAV-23410